subprojects/packagefiles/netstring-c/meson.build

```diff
@@ -0,0 +1,32 @@
+project(
+  'netstring-c',
+  'c',
+  version: '0.0.0+92104d3',
+  license : 'Unlicense',
+  meson_version: '>= 0.55',
+)
+
+# libm (glibc)
+cc = meson.get_compiler('c')
+libm_dep = cc.find_library('m')
```

Comment:
Shouldn't this be `required : false`?

Reply:
Seems to be required by the library: https://github.com/PeterScott/netstring-c/blob/92104d314ad885fd444b70bffd12a2963ef37a2f/makefile#L2

Reply:
That actually looks like an upstream bug as well. Most libc implementations don't have a separate libm, it's a quirk of glibc (which they plan to remove in the future i think, but with music to keep things working for people hard coding it). By hard coding it, we prevent this from building with a different libc, even on Linux, since i think only glibc and musl provide a libm

Reply:
My experience with C is very limited and what you're saying makes sense, adjusted accordingly.